posts/newmovie.pug

```diff
@@ -0,0 +1,44 @@
+extends Application
+
+block content
+	.container
+		.row
+			.col-xs-8.offset-xs-2
+				h1 new movie
+				hr
+				form(action="/newmovie" method="POST")
+					.form-group
+						label(for="movie-title") title
+						input.form-control(name="title" type="string" id="movie-title")
+					.form-group
+						label(for="movie-poster") posterUrl
+						input.form-control(name="coverImageURL" type="string" id="movie-poster")
+					.form-group
+						label(for="movie-trailer") trailer
+						input.form-control(ame="trailer" type="string" id="movie-trailer")
+					.form-group
+						label(for="movie-slug") slug
+						input.form-control(ame="slug" type="string" id="movie-slug")
+					.form-group
+						label(for="movie-director") director
+						input.form-control(name="directorName" type="string" id="movie-director")
+					.form-group
+						label(for="movie-starring") starring
+						input.form-control(ame="actorNames" type="string" id="movie-starring")
```

Comment:
you have typo in all your inputs names, dangerous type form wont work as intended. It should be name attribute not ame